components/Sidebar.tsx

```diff
@@ -1,11 +1,14 @@
-const Sidebar = () => {
+import { ReactElement, ReactNode } from "react";
+
+type Params = {
+  children?: ReactNode;
+}
+
+const Sidebar = ({ children }: Params):ReactElement => {
   return (
-    <div className="w-1/5 bg-stone-900 text-white border-r-2">
-      <ul className="list-none flex flex-col items-center gap-3 pt-5">
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">Home</li>
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">About</li>
-        <li className="hover:text-stone-400 cursor-pointer border-b-2 border-stone-400">Contact</li>
-      </ul>
+    <div className="w-64 bg-[#222628] text-white flex flex-col border-[1px] border-[#a5a19950]">
+      <h1 className="px-5 py-4 mb-3 font-bold text-lg tracking-wide cursor-pointer border-b-[1px] border-[#a5a19950]">Quick Links</h1>
+      {children}
```

Comment:
Move the title to the props, make it optional, and set the default value to `Quick Links`